config/pipelines/Jenkinsfile-contra-lib-sample/JenkinsfilePRBuild

```diff
@@ -0,0 +1,86 @@
+
+def libraries = ['contra-lib': ['master', 'https://github.com/openshift/contra-lib.git']]
+
+libraries.each { name, repo ->
+    library identifier: "${name}@${repo[0]}",
+            retriever: modernSCM([$class: 'GitSCMSource',
+                                  remote: repo[1]])
+
+}
+
+properties(
+        [
+                buildDiscarder(logRotator(artifactDaysToKeepStr: '30', artifactNumToKeepStr: '', daysToKeepStr: '90', numToKeepStr: '')),
+                disableConcurrentBuilds(),
+                parameters(
+                        [
+                                string(defaultValue: "${env.ghprbActualCommit}", description: '', name: 'ghprbActualCommit'),
+                                string(defaultValue: "${env.ghprbGhRepository}", description: '', name: 'ghprbGhRepository'),
+                                string(defaultValue: '', description: '', name: 'ghprbPullId'),
+                                string(defaultValue: '', description: '', name: 'ghprbPullAuthorLogin'),
+                                string(defaultValue: "${env.OPENSHIFT_NAMESPACE}", description: 'Project namespace for Openshift operations', name: 'OPENSHIFT_NAMESPACE'),
+                                string(defaultValue: "${env.DOCKER_REPO_URL}", description: 'The internal docker registry url', name: 'DOCKER_REPO_URL'),
+                                string(defaultValue: 'stable', description: 'The rpmbuild image tag', name: 'RPMBUILD_TAG'),
+                                string(defaultValue: 'stable', description: 'The cloud-image-compose image tag', name: 'CLOUD_IMAGE_COMPOSE_TAG'),
+                                string(defaultValue: 'stable', description: 'The singlehost-test image tag', name: 'SINGLEHOST_TEST_TAG'),
+                                string(defaultValue: CANNED_CI_MESSAGE, description: 'The ci message to build from', name: 'CI_MESSAGE')
```

Comment:
In the other file, this was ${CANNED_CI_MESSAGE}, not sure if one works and the other doesn't, or if both do, but probably best to be consistent